optuna/_study_summary.py

```diff
@@ -77,3 +91,33 @@ def __le__(self, other: Any) -> bool:
             return NotImplemented
 
         return self._study_id <= other._study_id
+
+    @property
+    def direction(self) -> StudyDirection:
+
+        if len(self._directions) > 1:
+            raise RuntimeError(
+                "This attribute is not available during multi-objective optimization."
+            )
+
+        return self._directions[0]
+
+    @direction.setter
+    def direction(self, d: StudyDirection) -> None:
+
+        if len(self._directions) > 1:
+            raise RuntimeError(
+                "This attribute is not available during multi-objective optimization."
+            )
+
+        self._directions = [d]
```

Comment:
I'm not sure if `StudySummary` needs setter methods. Do we want to overwrite the attributes?

Reply:
Oh, you are right. I remove the setter methods.